src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map(entries => entries.split(':')
+      .map(property => property.trim()));
+
+  const callback = (prev, [key, value]) => {
+    return Object.assign(prev, { [key]: value });
+  };
+
+  const stylesObject = stylesArray
+    .filter(entry => entry.length > 1)
+    .reduce(callback, {});
```

Comment:
```suggestion
  const filteredStylesArray = stylesArray.filter(entry => entry.length > 1);
  const stylesObject = Object.fromEntries(filteredStylesArray);
```

Reply:
brilliant use of `Object.fromEntries`, saves from pain of writing a reduce callback function. 
but i don't think you need `stylesObject` here. just immediately return `Object.fromEntries(filteredStylesArray)`

Another thing could be chaining the filter method to `stylesArray`